src/components/PuzzleTitle/PuzzleTitle.js

```diff
@@ -0,0 +1,34 @@
+import { h } from 'preact';
+
+const textShadow = '0 1px 4px rgba(0, 0, 0, 0.15)';
+const margin = '0 0.3em 0 0;';
+
+const PuzzleTitle = ({ index, title }) => (
+    <p>
+        <span className="puzzle-index">#{index}</span>
+        <span className="puzzle-title">{ title }</span>
+
+        <style jsx>{`
+            p {
+                font-size: 1.5em;
+            }
+
+            .puzzle-index {
+                color: #badece;
+                text-shadow: ${textShadow};
+                margin: ${margin};
+                vertical-align: middle;
+            }
+
+            .puzzle-title {
+                color: #f8d940;
```

Comment:
Should it be a part of `/src/styles/colour-palette.js` ?

Reply:
my plan is to address all issues of this kind in scope of #20

Reply:
👍